packages/react-native-reanimated/Common/cpp/reanimated/NativeAnimations/NativeAnimationValue.cpp

```
@@ -0,0 +1,48 @@
+#include <reanimated/NativeAnimations/NativeAnimationValue.h>
+
+#include <algorithm>
+#include <cmath>
+#include <type_traits>
+
+namespace reanimated::native_animation {
+
+bool valueMatchesTarget(const AnimationValue &value, const AnimationTarget target) {
```

Cppcheck (2.21.0)

[style] The function 'valueMatchesTarget' is never used.

(unusedFunction)